src/rigid_pendulum_poc/solver.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
```

Comment:
While I like `#pragma once`, we need to make sure that it is supported on all platforms that we'll be targeting

Reply:
We've considered using pragma once in our stk code-base. So far we haven't, only because it isn't technically standard, last time we checked. It does seem to be supported by every platform we know of though, so it's probably safe. But we're pretty cautious so we haven't adopted it yet. I could go either way in this case.